metricbeat/helper/prometheus/prometheus.go

```diff
@@ -119,6 +119,9 @@ func (p *prometheus) GetProcessedMetrics(mapping *MetricsMapping) ([]common.MapS
 	for _, family := range families {
 		for _, metric := range family.GetMetric() {
 			m, ok := mapping.Metrics[family.GetName()]
+			if m == nil {
```

Comment:
How about combining the two if blocks into one?

Reply:
nice, thanks